pkg/ingestor/parser/common/graph_builder.go

```diff
@@ -93,3 +95,63 @@ func addMetadata(predicates *assembler.IngestPredicates, foundIdentities []Trust
 		v.IsVuln.Origin = srcInfo.Source
 	}
 }
+
+func GetIsDep(foundNode model.PkgInputSpec, relatedPackNodes []model.PkgInputSpec, relatedFileNodes []model.PkgInputSpec, justification string) (*assembler.IsDependencyIngest, error) {
```

Comment:
hmm, perhaps we can move these helpers to a separate file or package? Doesn't quite fit into the "graph_builder" category.

Reply:
Moved it for now in a separate file in the same package. Let me know if that's OK

Reply:
yea that's fine!